ruby-challenges.rb

```diff
@@ -17,14 +25,39 @@
 # Expected output: 76
 
 nums_array2 = [25, 17, 47, 11]
-# Expected output: 100
+#Expected output: 100
+def add_arr (num_arr)
+  num_arr.map do |value|
+    puts num_arr.sum
+  end 
+end
+
+add_arr nums_array1
+add_arr nums_array2
+
 
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
+class Bike 
+  def initialize(model_parameter)
+    @model= model_parameter
+    @wheels=2
+    @current_speed=0
+  end
+  def bike_info
+    "this bike is a #{@model} model, it has and is going #{@current_speed}mph"
+  end 
+  def pedal_faster num
+    @current_speed+= 10
+  end 
+  def brake num
+    @current_speed-= 5
+  end
+end
```

Comment:
This looks like the logic is there, but where are you showing your results expected for this problem?